### PR TITLE
docs: Add AboutUs page and PPP for Pranavi

### PR DESCRIPTION
**Pull request by pra-navi**
## What type of PR is this? (check all applicable)

-   [ ] Refactor
-   [ ] Feature
-   [ ] Bug Fix
-   [ ] Optimization
-   [X] Documentation Update

## Description

-   Describe your changes.

## Related Tickets & Documents

-   Related Issue #

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 75% and above for new changes._

-   [ ] Yes
-   [X] No, and this is why: [Visit the TP page](https://pra-navi.github.io/tp/)
-   [ ] No, there are UI changes that I'm unable to cover.
-   [ ] I need help with writing tests

### CODE REVIEW

File: docs/team/pra-navi.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Given below are my contributions to the project.
+
+* **New Feature**: To be added soon!
+    * What it does: To be added soon!
+    * Justification: To be added soon!
+    * Highlights: To be added soon!
+    * Credits: To be added soon!
+
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=jiakai-17&breakdown=true)
```
Comment (HugeNoob) [inline docs/team/pra-navi.md:23]:
```suggestion
* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=pra-navi&breakdown=true)
```

Hmm you don't seem to be jiakai-17 🤔

File: docs/team/pra-navi.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+* **Project management**:
+    * To be added soon!
+
+* **Enhancements to existing features**:
+    * To be added soon!
+
+* **Documentation**:
+    * User Guide:
+        * Added Admonitions to User Guide [\#2](https://github.com/AY2324S1-CS2103T-T10-2/tp/pull/2)
```
Comment (HugeNoob) [inline docs/team/pra-navi.md:33]: Cmiiw but this is Jiakai's addition right? Remember to change it to yours instead.

**Review by HugeNoob — CHANGES_REQUESTED** [review]

**Comment by HugeNoob** [comment]
Also, the link in your description seems to be broken.

**Review by HugeNoob — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -270,14 +270,14 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `* * *`  | event planner                              | create tasks to do                 | know what tasks I need to do in preparation for the event                         |
+| `* * *`  | event planner                              | view both lists on the same screen | compare the task list and contact list while using the GUI                        |
+| `* * *`  | event planner                              | delete a contact                   | remove contacts that I no longer need                                             |
+| `* * *`  | user                                       | find a person by name              | locate details of persons without having to go through the entire list            |
+| `* *`    | user                                       | hide private contact details       | minimize chance of someone else seeing them by accident                           |
+| `*`      | user with many persons in the address book | sort persons by name               | locate a person easily                                                            |
```
Comment (HugeNoob) [inline docs/DeveloperGuide.md:280]: These changes are not relevant to this PR. Would be good if you can place these in a branch instead.

File: docs/team/pra-navi.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+* **Documentation**:
+    * User Guide:
+        * Added the images for the user guide.
+    * Developer Guide:
+        * To be added soon!
+
+* **Community**:
+    * PRs reviewed by me can be found [here](https://github.com/AY2324S1-CS2103T-T10-2/tp/pulls?q=is%3Apr+reviewed-by%3Apra-navi).
+    * Forum issues that I have participated in can be found [here](https://github.com/nus-cs2103-AY2324S1/forum/issues?q=is%3Aissue+commenter%3Apra-navi+).    * Reported bugs and suggestions for other teams in the class: To be added soon!
+    * Some parts of the history feature I added was adopted by several other class mates: To be added soon!
```
Comment (HugeNoob) [inline docs/team/pra-navi.md:39]:
```suggestion
    * Forum issues that I have participated in can be found [here](https://github.com/nus-cs2103-AY2324S1/forum/issues?q=is%3Aissue+commenter%3Apra-navi+).    
    * Reported bugs and suggestions for other teams in the class: To be added soon!
    * Some parts of the history feature I added was adopted by several other class mates: To be added soon!
```

You're missing a line break for the "Reported bugs..." bullet point.
image

**Review by HugeNoob — CHANGES_REQUESTED** [review]

**Review by HugeNoob — APPROVED** [review]

**Review by jiakai-17 — APPROVED** [review]

**Pull request merged**